sdk/keyvault/keyvault-keys/test/utils/utils.common.ts

```diff
@@ -34,3 +34,36 @@ export async function assertThrowsAbortError(cb: () => Promise<any>): Promise<vo
     throw new Error("Expected cb to throw an AbortError");
   }
 }
+
```

Comment:
I thought Chai had something like `shouldThrow`. Check that out. Otherwise this is awesome anyway!

Reply:
Unfortuately (and please correct me if I'm wrong) `shouldThrow` will not catch an error thrown from an async method, only synchronous methods. I found chai-as-promised but didn't want to take a dependency just for one method so I shamelessly stole this from msAssert in core-http :smile:

Reply:
We have chai-as-promised in our rush ecosystem 🌞 give that a try

Reply:
TIL we already pull in chai-as-promised so I'll make a change to use that. Thanks @sadasant !

Reply:
Fixed!